src/Apps/Artist/Routes/CV/Components/ArtistCVGroup.tsx

```diff
@@ -37,7 +37,18 @@ const ArtistCVGroup: FC<ArtistCVGroupProps> = ({ artist, relay, title }) => {
   const nodes = extractNodes(artist.showsConnection)
 
   if (nodes.length === 0) {
-    return null
+    return (
+      <GridColumns>
+        <Column span={12}>
+          <Text variant="lg-display">{title}</Text>
+        </Column>
+        <Column span={8} start={4}>
+          <Text variant="sm-display">
+            This dumb artist has no {title.toLowerCase()} yet.
```

Comment:
@jonallured - not sure about this message!

Reply:
LOLLLLLLL omg come on Jon

Reply:
This also illustrates the importance of strings in tests and flakiness (see previous PR comment from yesterday)